--- a/src/ntp/client.c	2025-05-07 12:02:18.008268405 +0100
+++ b/src/ntp/client.c	2025-05-07 17:28:34.987885445 +0100
@@ -240,7 +240,7 @@ static bool settime_skew(const double of
 	tx.offset = 1000000 * offset;
 	tx.modes = ADJ_OFFSET_SINGLESHOT;
 
-	log_debug(DEBUG_NTP, "Gradually adjusting system time by %ld us", tx.offset);
+	log_debug(DEBUG_NTP, "Gradually adjusting system time by %lld us", tx.offset);
 
 	if(adjtimex(&tx) < 0)
 	{
